### PR TITLE
Onboarding Process

### PR DESCRIPTION
**Pull request by FStop**

### CODE REVIEW

**Comment by FStop** [comment]
@danielbachhuber can you give this a first look and see if I'm headed in the right direction, missing anything obvious, etc?

File: team-culture/onboarding.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Onboarding Process
+
+Starting a new job can be overwhelming. We want to make your first days and weeks at Fusion as productive and stress-free as possible.
```
Comment (danielbachhuber) [inline team-culture/onboarding.md]: How do you feel about framing it in this structure:

1. Before you start
1. Your first day
1. Your first week
1. Your first month

Comment (FStop) [inline team-culture/onboarding.md]: I like it.

**Comment by danielbachhuber** [comment]
> can you give this a first look and see if I'm headed in the right direction, missing anything obvious, etc?

Looking pretty good, although I proposed an alternate structure I think might be more digestable. 

Want to throw this in a Google Doc? Google Docs might be more amenable to the editing that needs to be done.

**Comment by FStop** [comment]
Thanks. Starting with a Google Doc for editing sounds like a great idea. I'll set that up shortly.

**Comment by FStop** [comment]
[Here's a Google Doc](https://docs.google.com/a/fusion.net/document/d/1MOkIzcZnNS8pOR2VoplIlkWLSeUBumDDEHGMVT4i000/edit?usp=sharing) with the current state, editable by anyone in Fusion network.

**Comment by danielbachhuber** [comment]
:+1:

**Pull request merged**